src/components/FilterTags/FilterTags.js

```diff
@@ -145,7 +146,15 @@ const FilterTags = () => {
       <input
         type="text"
         className="p-filter-tags__input"
-        onClick={() => setFilterPanelVisibility(true)}
+        onClick={() => {
+          setFilterPanelVisibility(true);
+          if (filterPanelVisibility) {
```

Comment:
The previous line you're setting the `filterPanelVisibility` to `true` so this check is not necessary as it should always be called.